### ISSUE TITLE
Silvercrest Binding

### ISSUE THREAD

**Issue by xarbit** [giu1io/homebridge-ankuoo-rec-switch#6]
I see that openHab has support for the two:

    Lidl WI-FI Steckdose SWS-A1 SilverCrest IAN 103043
    EasyHome WIFI Steckerset von Aldi-Süd und FHEM

you mentioned in your blog post [here](http://blog.giuliomontagner.com/post/155085663708/trying-to-hack-ankuoo-rec-switch-and-make-it)
I wonder what it would take to add support for the Ankuoo's? I have a Malmbergs WiFI Switch Module (CSW201) which is the same build as the Ankuoo REC and even uses the Ankuoo REC iOS/android App. 

Here is the Binding. 
https://docs.openhab.org/addons/bindings/silvercrestwifisocket/readme.html

What do you think?

Thanks.

Jason

**Comment by giu1io** [comment]
Adding the support for the Ankuoo should be straightforward once you know the encryption key that’s used to “secure” the payload. I looked over the source code for the OpenHab binding and they have embedded “0123456789abcdef” as the encryption key. As I have said in my blog post I have tried that key but it didn’t work for me. That binding should work perfectly with the Ankuoo once the correct key is discovered and replaced.

**Comment by xarbit** [comment]
thanks, now I understand.. I was able to follow and figure out the encrypted payload.. 
I used a slightly different approach to get to it (openwrt router + tcpdump + wireshark) .. 

Now I do have my ON/OFF payloads and I successfuly tested them with PacketSender.

now I need to figure out the best way to integrate it in openHAB, without the key it dont make sense to work on a real binding. 

Anyway... 
Thank's alot for posting the useful information on your blog!
Helped me alot ..
